onnxruntime/python/tools/transformers/models/stable_diffusion/requirements-cuda11.txt

```diff
@@ -1,7 +1,7 @@
 -r requirements.txt
 
```

Comment:
The file name should be exactly "requirements.txt", no extra words. So that Github's dependency bot and Microsoft's Component Governance can recognize.

Reply:
You may put such files in different folders to distinguish